### ISSUE TITLE
Don't flag docblock comments as incorrectly indented

### ISSUE THREAD

**Issue by paulschreiber** [WordPress-Coding-Standards/WordPress-Coding-Standards#301]
For functions indented one tab: in a docblock comment, PHPCS (with WordPress-VIP) is flagging tab-space-star as incorrectly indented. It wants tab-tab-star, which @aliso tells me is incorrect.

```
	/**
	 * Callback function for foobar action. Dances a jig.
	 */
	public static function foobar() {
```

**Comment by westonruter** [comment]
I cannot reproduce. Given a test file `class-test.php`:

```php
<?php

class Test {
	/**
	 * Callback function for foobar action. Dances a jig.
	 */
	public static function foobar() {
		baz();
	}
}
```

Running the WordPress-VIP tests shows:

```
$ phpcs --standard=WordPress-VIP -v class-test.php
Registering sniffs in the WordPress VIP standard... DONE (39 sniffs registered)
Creating file list... DONE (1 files in queue)
Changing into directory /Users/thunderboy/Projects/vip-quickstart
Processing class-test.php [PHP => 46 tokens in 13 lines]... DONE in 3ms (0 errors, 0 warnings)
```

Is your PHP_CodeSniffer checked out to 2.0? Do you have the latest commit https://github.com/squizlabs/PHP_CodeSniffer/commit/7ba71b47a1f5da02a4b98a7c142e0fbaa0a647e8? Is your WordPress-Coding-Standards also checked out at the latest `master` or `develop`?

**Comment by westonruter** [comment]
Closed due to lack of reporter feedback and cannot reproduce.
